.travis.yml

```diff
@@ -6,7 +6,8 @@ install:
       - make setup
       - make update
 before_script:
-      - make lint
+      - make lint-python
+      - make lint-js
```

Comment:
While developing, we run `make lint`, which runs `make lint-js-fix` to automatically fix some JS linting errors. We don't want to make these fixes in Travis, so we just run `make lint-js`.